Sources/Widgets/Widgets3D/SplineWidget/example/controlPanel.html

```diff
@@ -54,6 +54,48 @@
       <input class="freehandDistance" type="range" min="0.05" max="1.0" step="0.05" value="0.2">
     </td>
   </tr>
+  <tr>
+    <td>Closed spline</td>
+    <td></td>
+    <td>
+      <input class="close" type="checkbox" checked />
+    </td>
+  </tr>
+  <tr>
+  <td>Boundary conditions</td>
+  <td></td>
+  <td>
+    <select class="boundaryCondition">
+      <option value="0">default</option>
```

Comment:
interestingly, default is "0" here, it probably means that 0 should be used instead of null in Spline1D/Constants.js